src/xrEngine/EngineAPI.cpp

```diff
@@ -39,13 +39,15 @@ static bool r2_available = false;
 //////////////////////////////////////////////////////////////////////
 
 void __cdecl dummy(void) {}
+constexpr auto dummyCreate  = [](auto){ return nullptr; };
+constexpr auto dummyDestroy = [](auto){ };
```

Comment:
It's better to make it plain functions. Also, destroy should assert that passed pointer is nullptr.

Reply:
Nope. This will crash when trying to release pGamePersistent with `-nogame` option